sprint8/tema2.js

```diff
@@ -0,0 +1,13 @@
+function sumArray(numbers) {
+    return numbers.reduce((sum, num) => sum + num, 0);
+}
+
+// Example 1
+let array1 = [1, -5, 20, -34, 16, 29, 36, -4];
+// Output: 59
+console.log(sumArray(array1)); // Output: 59
```

Comment:
```suggestion
console.log(sumArray(array1));
```
Se poate sterge comentariul de la final